api.go

```diff
@@ -228,9 +228,15 @@ func RecoverCluster(conf *Config, fsm FSM, logs LogStore, stable StableStore,
 		return err
 	}
 
-	// Sanity check the Raft peer configuration.
-	if err := checkConfiguration(configuration); err != nil {
-		return err
+	// Sanity check the Raft peer configuration. Note that it's ok to
+	// recover to an empty configuration if you want to keep the data
+	// but not allow a given server to participate any more (you'd have
+	// to recover again, or add it back into the existing quorum from
+	// another server to use this server).
```

Comment:
@ongardie-sfdc agreed - I was just having misgivings about this as I was grabbing lunch and noticed your comment as I was jumping on to remove this :-)

I was thinking the user had signed up for this because they removed the server from the cluster, but if it comes back and somehow gets elected as leader it can cause damage. I'll remove this in a separate PR.